cmd/manager/main.go

```diff
@@ -19,14 +19,19 @@ package main
 import (
 	"flag"
 	"log"
+	"os"
 	"time"
 
 	configv1 "github.com/openshift/api/config/v1"
 	machinev1 "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
 	"github.com/openshift/machine-api-operator/pkg/metrics"
 	"k8s.io/klog/v2"
+	"k8s.io/klog/klogr"
```

Comment:
```suggestion
	"k8s.io/klog/klog"
```

This is causing your builds to fail

Reply:
You will have to re-vendor most likely as well

Reply:
yes all builds were failing because of vendoring. Latest push should fix that.

Reply:
yes all builds were failing because of vendoring. shoudl be fixed now